apps/site/src/app/(main)/(home)/sections/Landing/Landing.tsx

```diff
@@ -1,41 +1,37 @@
 "use client";
 
 import { Suspense } from "react";
-import Image from "next/image";
 import { PerspectiveCamera } from "@react-three/drei";
 
 import hasDeadlinePassed from "@/lib/utils/hasDeadlinePassed";
+import haveApplicationsOpened from "@/lib/utils/haveApplicationsOpened";
 import View from "@/components/canvas/View";
 import Button from "@/lib/components/Button/Button";
-import Fireflies from "../../components/Fireflies";
-
-import fogLeft from "@/assets/images/fog-left.png";
-import fogRight from "@/assets/images/fog-right.png";
 
 import styles from "./Landing.module.css";
 
 const Landing = () => {
 	const deadlinePassed = hasDeadlinePassed();
+	const applicationsOpened = haveApplicationsOpened();
 
 	return (
 		<section className={styles.landingBackground}>
 			<View className="absolute w-full h-full">
 				<Suspense fallback={null}>
-					<Fireflies />
 					<PerspectiveCamera makeDefault position={[0, -0.1, 1]} />
 				</Suspense>
 			</View>
 			<div className="flex flex-col justify-center items-center min-h-screen text-center overflow-x-hidden relative">
-				<Image src={fogLeft} alt="Fog" className={styles.fogLeft} fill />
-				<Image src={fogRight} alt="Fog" className={styles.fogRight} fill />
-				<h1 className="font-display text-4xl md:text-5xl font-bold mb-2">
-					IrvineHacks 2024
+				<p className="font-display text-2xl md:text-3xl">January 24&ndash;26</p>
+				<h1 className="font-display text-4xl md:text-5xl font-bold mb-5">
+					IrvineHacks 2025
 				</h1>
-				<p className="font-display text-2xl md:text-3xl">January 26&ndash;28</p>
 				{deadlinePassed ? (
 					<Button className="z-10" text="Applications have closed!" disabled />
-				) : (
+				) : applicationsOpened ? (
 					<Button className="z-10" text="Apply" href="/apply" />
+				) : (
+					<Button className="z-10" text="Coming Soon..." disabled />
```

Comment:
note: the hover still appears to be on however you can't click on the button

Reply:
Graphics wanted the hover effect for the buttons, which I left for disabled buttons. I believe this is fine functionality wise.

Reply:
yes, functionality is fine